index.html

```diff
@@ -218,6 +218,14 @@
           capture to produce silence, nor does hiding a media element cause
           captured video to stop.
         </p>
+        <p>
+          Captured audio from an element with
+          an <a href="http://www.w3.org/TR/html5/embedded-content-0.html#effective-playback-rate">effective
+          playback rate</a> other than 1.0 MUST be altered in pitch if
```

Comment:
This is a bit confusing. Do you mean to alter the pitch after the resampling so that it's effectively time-stretching, or do you mean that this playbackRate change is simply resampling, so that the pitch is changed ?